### ISSUE TITLE
Errata: Section 8.5.2, BatchNorm1D -> BatchNorm1d

### ISSUE THREAD

**Issue by rasbt** [deep-learning-with-pytorch/dlwpt-code#46]
I think it should be "BatchNorm1d":
![Screen Shot 2021-01-15 at 5 46 57 PM](https://user-images.githubusercontent.com/5618407/104789153-d0b28b80-5759-11eb-9c33-8a063de3743b.png)
